cmd/proxy-server/main_test.go

```diff
@@ -170,14 +175,36 @@ func TestVolumesHandler(t *testing.T) {
 			getRolesFn := func(ctx context.Context) (*roles.JSON, error) {
 				return &rff, nil
 			}
-			svc := role.NewService(fakeKube{GetConfiguredRolesFn: getRolesFn}, successfulValidator{})
+			roleSvc := role.NewService(fakeRoleKube{GetConfiguredRolesFn: getRolesFn}, successfulRoleValidator{})
+
+			// create storage client
+			storageClient := &mockStorage.FakeStorageServiceClient{
+				GetPowerflexVolumesFn: func(context.Context, *pb.GetPowerflexVolumesRequest, ...grpc.CallOption) (*pb.GetPowerflexVolumesResponse, error) {
+					// Create fake volume details
+					mockVolumeDetails := []*pb.Volume{
+						{
+							Name:     "k8s-6aac50817e",
+							Size:     8,
+							SystemId: "542a2d5f5122210f",
+							Id:       "volumeId1",
+							Pool:     "pool1",
```

Comment:
This is nitpicking because the tests pass, but the volume details here don't quite align with the data in redis. In this test, the pool should be `bronze`. Other tests have similar patterns.